govcd/orgvdcnetwork.go

```diff
@@ -89,8 +89,8 @@ func (orgVdcNet *OrgVDCNetwork) Delete() (Task, error) {
 
 // Looks for an Org Vdc network and, if found, will delete it.
 func RemoveOrgVdcNetworkIfExists(vdc Vdc, networkName string) error {
-	network, err := vdc.FindVDCNetwork(networkName)
-	if err == nil && network != (OrgVDCNetwork{}) {
+	network, err := vdc.GetOrgVdcNetworkByName(networkName, false)
+	if err == nil {
```

Comment:
Wouldn't this "mask" the `err`? If the error was not nil - it would not try to remove and return`nil` I think while in fact there could have been some error during deletion.

Reply:
Added more error handling, and an explicit test